cdhweb/projects/models.py

```diff
@@ -290,6 +290,17 @@ def alums(self):
             .exclude(membership__in=self.current_memberships()) \
             .order_by("last_name")
 
+    def get_sitemap_urls(self, request):
+        """Override sitemap to prioritize projects built by CDH with a website."""
+        # output is a list of dict; there should only ever be one element. see:
+        # https://docs.wagtail.io/en/stable/reference/contrib/sitemaps.html#urls
+        urls = super().get_sitemap_urls(request=request)
+        if self.website_url and self.cdh_built:
+            urls[0]["priority"] = 0.7
+        elif self.website_url or self.cdh_built:
+            urls[0]["priority"] = 0.6
```

Comment:
should we also add priority for things that are highlighted?

Reply:
that makes sense to me. should we also maybe prioritize staff profiles (vs e.g. grad fellows)? trying to think of other uses for it. if we do add more priority to projects so they can have up to 0.8, I wonder if we should give more to featured blogposts too so that they're not still at 0.6 relative to projects.

Reply:
it might even just be simpler to only bump priority slightly for highlighted projects and remove the current logic...i think highlighted and cdh-built are pretty much 1:1 and i'm not sure how important website URL is

Reply:
What if we de-prioritize projects without a url? A lot of the older projects that never got far enough along to get a url are kind of vapour ware in a way. (But might not apply to working groups...)  

Not all cdh-built projects are highlighted (although agree there is substantial overlap; but see e.g. the New NLP project); it's more current highlights / current work, I think. 

I agree that setting them to 0.8 seems too high. Maybe we're overcomplicating it — we can set to current logic and make a chore to investigate revising, if you think it's worth a little more thought.

Reply:
that sounds good to me – I'll make a chore and we can leave it this way for now. definitely seems like there are a lot of things to consider.